src/main/java/org/broadinstitute/hellbender/tools/dataflow/pipelines/DataflowReadsPipeline.java

```diff
@@ -103,26 +105,29 @@ final protected void setupPipeline(Pipeline pipeline) {
     }
 
     @VisibleForTesting
-    protected PCollection<?> applyTransformsToPipeline(String headerString, PCollection<Read> preads) {
-        for (ReadFilter filter : getReadFilters()) {
+    protected PCollection<?> applyTransformsToPipeline(final String headerString, final PCollection<Read> preadsArgs) {
```

Comment:
the previous code assigned to a method parameter which is bad style. I renamed it now to 'preadsIn' to not be confused with 'arguments'